cmd/src/campaign_plans_create_from_patches.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/mattn/go-isatty"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	usage := `
+Create a campaign plan from a set of patches (in unified diff format) to repository branches.
+
+Stdin is expected to be a JSON array of {repository: string, baseRevision: string, patch: string}. The repository value is the repository's GraphQL ID (which you can look up given a repository name using 'src repos get -name=...').
```

Comment:
Sorry, but I have to:

```suggestion
STDIN is expected to be a JSON array of {repository: string, baseRevision: string, patch: string}. The repository value is the repository's GraphQL ID (which you can look up given a repository name using 'src repos get -name=...').
```

Reply:
Or, if the issue is clarity rather than aesthetics, you could spell it out: "Standard input is expected…"

Since this is documentation for humans that wouldn't be so bad. 😁

Reply:
In my case it was purely aesthetics 😉

Reply:
Fair enough! (On aesthetics I'd vote for the original capitalization, personally—"stdin" isn't an acronym)